src/pages/Movies.jsx

```diff
@@ -19,26 +19,48 @@ const Movies = ({ templateRef }) => {
 
     const filterButtonsRef = useRef(null);
 
-    const [searchParams] = useSearchParams();
+    const [searchParams, setSearchParams] = useSearchParams();
 
     const filter = searchParams.get('filter') || null;
+    const sortby = searchParams.get('sortby') || '';
     const order = searchParams.get('order') || null;
 
+    const [selectedSortBy, setSelectedSortBy] = useState(sortby);
```

Comment:
Hey @danielholmes, just wanted to check, do I need to define 'selectedSortBy' as state?  Or would it be better to get my select dropdown on line 232 working as an uncontrolled component?

Reply:
@mbborromeo  I don't know how you'd do what you're trying to do as an uncontrolled component.